Validation/HcalDigis/src/HcalDigisValidation.cc

```diff
@@ -957,11 +795,9 @@ template<class Digi> void HcalDigisValidation::reco(const edm::Event& iEvent, co
 
         // SimHits once again !!!
         double eps = 1.e-3;
-        double ehits = 0.;
-        double ehits1 = 0.;
-        double ehits2 = 0.;
-        double ehits3 = 0.;
-        double ehits4 = 0.;
+	std::vector<double> v_ehits;
```

Comment:
@kencall - this is better written as

std::vector<double> v_ehits(maxDepth_[isubdet],0.);